crates/aliveness/src/lib.rs

```diff
@@ -0,0 +1,84 @@
+use std::{
+    net::{IpAddr, Ipv4Addr, SocketAddrV4},
+    time::Duration,
+};
+
+use color_eyre::eyre::{Result, WrapErr};
```

Comment:
I converged to each time I'm authoring a new library crate, I'm trying to use `thiserror` instead of `color_eyre` since this is more compatible and allows the consuming crates of your crate to handle errors selectively. Have a look e.g. in the `communication` crate for examples. If that is too much effort, feel free to discuss.

Reply:
Alright, done.